documentation/modules/ref-standard-infra-config-fields.adoc

```diff
@@ -10,42 +10,50 @@
+|`broker.podTemplate.spec.priorityClassName` |Specifies the priority class to use for the broker Pod so you can prioritize broker Pods over other Pods in the {KubePlatform} cluster.
+|`broker.podTemplate.spec.affinity` |Specifies the affinity settings for the broker Pod so you can specify where on particular nodes a Pod runs, or if it cannot run together with other instances.
+|`broker.podTemplate.spec.tolerations` |Specifies the toleration settings for the broker Pod, which allow this Pod to run on certain nodes on which other Pods cannot run.
+|`broker.podTemplate.spec.containers.initialDelaySeconds` |Specifies probe delay value in seconds for the broker Pod.
+|`broker.podTemplate.spec.containers.timeoutSeconds` |Specifies probe timeout value in seconds for the broker Pod.
+|`broker.podTemplate.spec.containers.resources` |Specifies broker Pod resource requests and limits for CPU and memory.
+|`broker.podTemplate.spec.containers.env` |Specifies environment variables for the broker Pod.
+|`broker.connectorIdleTimeout` |Specifies the AMQP idle timeout to use for connection to router.
+|`broker.connectorWorkerThreads` |Specifies the number of worker threads of the connection to the router.
+|`broker.storageClassName` |Specifies what storage class to use for the persistent volume for the broker.
+|`broker.updatePersistentVolumeClaim` |If the persistent volume supports resizing, setting this value to `true` allows the broker storage to be resized.
+|`router.resources.memory` |Specifies the amount of memory allocated to the router.
+|`router.linkCapacity` |Specifies the default number of credits issued on AMQP links for the router.
+|`router.handshakeTimeout` |Specifies the amount of time in seconds to wait for the secure handshake to be initiated.
+|`router.minReplicas` |Specifies the minimum number of router Pods to run; a minimum of two are required for high availability (HA) configuration.
+|`router.podTemplate.metadata.labels` |Specifies the labels added to the router Pod.
+|`router.podTemplate.spec.priorityClassName` |Specifies the priority class to use for the router Pod so you can prioritize router pods over other pods in the {KubePlatform} cluster.
+|`router.podTemplate.spec.affinity` |Specifies the affinity settings for the router Pod so you can specify where on particular nodes a pod runs, or if it cannot run together with other instances.
+|`router.podTemplate.spec.tolerations` |Specifies the toleration settings for the router Pod, which allow this Pod to run on certain nodes on which other Pods cannot run.
+|`router.podTemplate.spec.containers.initialDelaySeconds` |Specifies probe delay value in seconds for the router Pod.
+|`router.podTemplate.spec.containers.timeoutSeconds` |Specifies probe timeout value in seconds for the router Pod.
+|`router.podTemplate.spec.containers.resources` |Specifies router Pod resource requests and limits for CPU and memory.
```

Comment:
@k-wall, Can you please confirm that this description is correct (and applicable for both broker and router)?

Reply:
you've missed level for the two probe settings from the dotted description:  spec.containers.readinessProbe.., spec.containers.livenessProbe..